src/IO/H5/Python/VolumeData.cpp

```diff
@@ -0,0 +1,71 @@
+
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+#include <hdf5.h>
```

Comment:
what is `hdf5.h` being used for? I feel like this one should be something that is handled by the H5 wrappers.

Reply:
Removed